.vib/node/goss/vars.yaml

```diff
@@ -0,0 +1,29 @@
+binaries:
+  - corepack
+  - node
+  - npm
+  - npx
+  - node-gyp
+  - python
+  - yarn
+files:
+  - paths:
+      - /opt/bitnami/node/lib/node_modules/corepack/package.json
+      - /opt/bitnami/node/lib/node_modules/node-gyp/package.json
+      - /opt/bitnami/node/lib/node_modules/npm/package.json
+      - /opt/bitnami/node/lib/node_modules/yarn/package.json
+directories:
+  - mode: "0755"
+    paths:
+      - /app
+      - /opt/bitnami/node/bin
+      - /opt/bitnami/node/lib/node_modules
```

Comment:
I would say you are already implicitly verifying these dirs exist by checking the different binaries and the list of modules.